build-logic/project-plugins/src/main/kotlin/com.hedera.hashgraph.jpms-module-dependencies.gradle.kts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2016-2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id("org.gradlex.java-module-dependencies") }
+
+javaModuleDependencies {
+    warnForMissingVersions.set(false) // do not expect versions in catalog
+
+    // The following is not working because it looks for 'swirlds.test.framework'
```

Comment:
Is this commented code still needed?

Reply:
Yes. The commentYes. The comment continues on the next lines and explains a piece of configuration – `moduleNamePrefixToGroup.put("com.", "com.swirlds")` – that would make the configuration more compact but which does not work (right now) as explained in the comment – issue with`-` (used as separator in project/folder names) vs `.` (used as separator in module names).

Maybe it's mostly for me though. I would like to improve this, but right now I don't have a better solution.